pkg/migrate/cstor/volume.go

```diff
@@ -606,9 +629,13 @@ func (v *VolumeMigrator) createTmpSC(scName string) (*storagev1.StorageClass, er
 			Annotations: scObj.Annotations,
 			Labels:      scObj.Labels,
 		}
+		tmpSCObj.Annotations["pv-name"] = v.PVName
 		tmpSCObj, err = v.KubeClientset.StorageV1().
 			StorageClasses().Create(tmpSCObj)
 		if err != nil {
+			if k8serrors.IsAlreadyExists(err) {
+				return nil, err
```

Comment:
error should be nil here , since we want to skip the creation of temp-SC and continue the other migration steps ?

Reply:
ok ,ignore the above comment,  return err is correct, which will be returned further from the caller.